### DISCUSSION TITLE
Information about item to be delete

### DISCUSSION THREAD

**Discussion by jmachado-virt** [community/community#19824] [Projects and Issues] [answered]
Before the confirmation of the deletion of a item, it would be nice that could be provided information about the item that will be deleted. This could avoid the deletion of the wrong element on a board.

**Comment by themiralay** [comment, marked as answer]
[This present issue](https://github.com/github-community/community/discussions/7575) is probably most popular issue with regard to tasks in Github Projects (Beta)!
 
 Other (possible) duplicates are:
 
 * [[Projects Beta] Show number of tasks vs completed task after issue title #16616](https://github.com/orgs/github-community/discussions/16616)
 * [[Projects Beta] Show x of n tasks on issue #13033](https://github.com/orgs/github-community/discussions/13033)
 * [[Projects beta] Task list progress #16721](https://github.com/orgs/github-community/discussions/16721)
 * [Show percent tasks complete in a new projects Beta field #12050](https://github.com/orgs/github-community/discussions/12050)
